Simplified/Reader2/UI/NYPLUserSettingsVC.swift

```diff
@@ -50,63 +50,54 @@ import R2Navigator
     fatalError("init(coder:) not implemented")
   }
 
-  override func loadView() {
-    let view = NYPLReaderSettingsView(width: 300)
-    view.delegate = self
-
-    view.colorScheme = userSettings.colorScheme
-    view.fontSize = userSettings.fontSize
-    view.fontFace = userSettings.fontFace
-    view.backgroundColor = userSettings.backgroundColor
-    view.publisherDefault = userSettings.publisherDefault
-    
-    self.view = view;
-  }
-
   override func viewDidLoad() {
     super.viewDidLoad()
-    self.preferredContentSize = self.view.bounds.size
+    
+    let view = NYPLReaderSettingsView(width: 330,
```

Comment:
not a big deal, but I was wondering why not leaving the view creation inside `loadView` (basically everything between lines 56 - 63), and instead of adding a subview you'll keep doing `self.view = view` (in loadView).